src/configuration/index.d.mts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @typescript-eslint/no-empty-object-type */
+export * from "./globals.mjs";
 
 /**
```

Comment:
Could you move everything below this line into a `src/configuration/documents.d.mts` file? Then write `export * from "./documents.mjs"`